pkg/sql/compile/compile.go

```diff
@@ -645,6 +646,20 @@ func (c *Compile) ConstructScope() *Scope {
 	return ds
 }
 
+func (c *Compile) ConstructMergeScope() *Scope {
+	ds := &Scope{Magic: Merge}
+	ds.Proc = process.NewWithAnalyze(c.proc, c.ctx, 1, c.anal.Nodes())
+	ds.Proc.LoadTag = true
+	ds.Proc.ParallelLoad = true
```

Comment:
remove the load related codes outside of ConstructMergeScope() or remove whole method.